### PR TITLE
Publish Aug 2019 relnotes to live docs

### PR DESCRIPTION
**Pull request by DawnWood-Docker**

### CODE REVIEW

File: _data/glossary.yaml [hunk 1]
```diff
@@ -219,6 +221,8 @@ parent image: |
   in the image's Dockerfile. All subsequent commands are applied to this parent
   image. A Dockerfile with no `FROM` directive has no parent image, and is called
   a **base image**.
+persistent storage: |
+  Persistent storage or volume storage provides a way for a user to add a persistent layer to the running container's file system. This persistent layer could live on the container host or an external device. The lifecycle of this persistent layer is not connected to the life cycle of the container, allowing a user to retain state.
```
Comment (lena-larionova) [inline _data/glossary.yaml]: This definition uses both "lifecycle" and "life cycle". They're both technically right, but can we decide on one?

Comment (traci-morrison) [reply _data/glossary.yaml]: Yes, it should be "lifecycle." @DawnWood-Docker Can we update this? Thanks.

File: ee/dtr/admin/configure/external-storage/storage-backend-migration.md [hunk 2]
```diff
@@ -18,50 +18,50 @@ Docker recommends the following steps for your storage backend and metadata migr
...
-  * For changes to reconfigure and restore options in DTR 2.6, see [docker/dtr reconfigure](/reference/dtr/2.6/cli/reconfigure/) and [docker/dtr restore](/reference/dtr/2.6/cli/restore). 
+  * For changes to reconfigure and restore options in DTR 2.6, see [docker/dtr reconfigure](/datacenter/dtr/2.6/reference/cli/reconfigure/) and [docker/dtr restore](/datacenter/dtr/2.6/reference/cli/restore/).
   * For Docker's recommended recovery strategies, see [DTR 2.6 lost tags after reconfiguring storage](https://success.docker.com/article/dtr-26-lost-tags-after-reconfiguring-storage).
-  * For NFS-specific changes, see [Use NFS](nfs.md). 
+  * For NFS-specific changes, see [Use NFS](nfs.md).
   * For S3-specific changes, see [Learn how to configure DTR with Amazon S3](s3.md).
 
-Upgrade to [DTR 2.6.4](#dtr-264-and-above) and follow [best practice for data migration](#best-practice-for-data-migration) to avoid the wiped tags issue when moving from one NFS serverto another. 
+Upgrade to [DTR 2.6.4](#dtr-264-and-above) and follow [best practice for data migration](#best-practice-for-data-migration) to avoid the wiped tags issue when moving from one NFS serverto another.
```
Comment (lena-larionova) [inline ee/dtr/admin/configure/external-storage/storage-backend-migration.md]: There's a space missing in "serverto"

File: _data/glossary.yaml (hunk 1 above)
Comment (DawnWood-Docker) [reply _data/glossary.yaml]: @traci-morrison yes

File: ee/ucp/admin/configure/external-auth/index.md [hunk 3]
```diff
@@ -225,9 +225,9 @@ LDAP directory.
 As of UCP 3.1.5, LDAP-specific `GET` and `PUT` API endpoints have been added to the Config resource. Note that swarm mode has to be enabled before you can hit the following endpoints:
 
 - `GET /api/ucp/config/auth/ldap` - Returns information on your current system LDAP configuration.
-- `PUT /api/ucp/config/auth/ldap` - Lets you update your LDAP configuration. 
+- `PUT /api/ucp/config/auth/ldap` - Lets you update your LDAP configuration.
 
-See [UCP API Documentation](/reference/ucp/3.1/api/) for additonal information.
+See [UCP API Documentation](/reference/ucp/3.2/api/) for additonal information.
```
Comment (lena-larionova) [inline ee/ucp/admin/configure/external-auth/index.md]: Typo in "additional"

**Pull request merged**